siac/main.go

```diff
@@ -160,6 +161,14 @@ func die(args ...interface{}) {
 	os.Exit(exitCodeGeneral)
 }
 
+// yesNo returns "Yes" if b is true, and "No" if b is false.
+func yesNo(b bool) string {
```

Comment:
I considered that as well, but my first assumption for `boolToString` would be `"true"` and `"false"`.